C++/straw.cpp

```diff
@@ -760,18 +760,18 @@ vector<contactRecord> readBlock(istream &fin, CURL *curl, bool isHttp, indexEntr
 
 // reads the normalization vector from the file at the specified location
 vector<double> readNormalizationVector(istream &bufferin, int version) {
-    long nValues;
+    long long nValues;
     if (version > 8) {
         bufferin.read((char *) &nValues, sizeof(long));
     } else {
         nValues = (long) readIntFromFile(bufferin);
     }
 
-    unsigned long numValues = static_cast<unsigned long>(nValues);
+    unsigned long long numValues = static_cast<unsigned long>(nValues);
```

Comment:
should it be `static_cast<unsigned long long>`?

Reply:
Fixed; thank you.